script/Ciel/Day 9/09_01_CIEL09_6B.txt

```diff
@@ -583,76 +683,89 @@ Weren't you and I supposed to be having some fun? Come on, why don't you give it
 [sha:a101a6e6b8a4ca6639adb49331e6ed226cf7a16e]{
 -- Page 648, Offset 23817.
 -- アルクェイドは苛立ちを踏み倒すように仇敵へと歩を進める。
-// I'm really unsure on that one
-As if to trample on the source of her irritation, Arcueid begins walking toward her sworn enemy.
+Arcueid starts walking toward her enemy,
+ as though she intends to stamp out the source of her frustration.
 }
 [sha:d9409eedcd69476368321ee96b66f96119a85305]{
 -- Page 648, Offset 23818.
 -- シエルに“本気になれ”と吸血鬼は糾弾した。
-The vampire lambasted Ciel, telling her to give it her all.
+The vampire disparaged Ciel, and demanded she stop holding back.
 }
 [sha:1b16b32af519ff3345c9d480b85c56ee0e02f735]{
 -- Page 648, Offset 23819.
 -- ……だが、それは大きな過ちだ。
-However, she was gravely mistaken.
+Except... she's misunderstood completely.
 }
 [sha:8f8e60ef28737e4ae896013687a6321a48642fd4]{
 -- Page 648, Offset 23820.
 -- 代行者は本気だった。現状の限られた装備内での話だったが、文字通り全力を尽くしたのだ。
-The Executor was giving it her all. Certainly, some allowances have to be made considering her limited arsenal, but she has quite literally pulled out all the stops.
+The Executor was already giving it her all.
+ Certainly, some allowances have to be made considering her limited arsenal, but she has quite literally pulled out all the stops.
```

Comment:
Keep with the theme you already established for 本気.

>The Executor wasn't holding back.

The 文字通り doesn't quite work in English...
The sentence is also far harder to grasp for some reason. Just rewrite.
>She gave it her very best, or at least as much as her currently limited arsenal would allow.